bin/ddg2logo

```diff
@@ -281,11 +285,16 @@ for fname in fnames:
 
 data = np.vstack(tuple(data))
 
-# set values above the maximum value to the maximum
-# move the whole range of values, so that values at maximum become 0
-# and invert sign so that higher values are more stabilizing
-data[ data >= options.threshold ] = options.threshold
-data = (data - options.threshold) * (-1)
+# set values above the threshold (for stabilizing mutations) or below
+# it (for the destabilizing ones) equal to 0
+# In the case of stabilizing mutations, invert sign so that higher values are more stabilizing
+
+if options.threshold_stab or options.threshold_stab == 0.0:
```

Comment:
I will change it. Just to add a bit of info here. I checked for 0s because when using the other way around (if options.threshold_stab:), there was the case in which if I set the threshold value to 0, it was'nt entering the if statement. In this way it left the data untouched. But sure I'll modify it immediately.